WordPress/Classes/ViewRelated/Stats/Insights/Posting Activity/PostingActivityDay.swift

```diff
@@ -48,7 +48,7 @@ private extension PostingActivityDay {
     }
 
     @IBAction func dayButtonPressed(_ sender: UIButton) {
-        dayButton.backgroundColor = WPStyleGuide.Stats.PostingActivityColors.orange
+        dayButton.backgroundColor = WPStyleGuide.Stats.PostingActivityColors.pink
```

Comment:
You could just access `.accent` here directly, instead of wrapping it in a more fragile color name?

Reply:
My preference would be to still access the colors via `WPStyleGuide.Stats.PostingActivityColors` (so they're still defined in one central location), but change the names to what they're used for instead of what color they are. 

For example: 
```
        struct PostingActivityColors {
            static let range1 = UIColor.neutral(shade: .shade10)
            static let range2 = UIColor.muriel(color: MurielColor(name: .airo, shade: .shade10))
            static let range3 = UIColor.muriel(color: MurielColor(name: .airo, shade: .shade30))
            static let range4 = UIColor.muriel(color: MurielColor(name: .airo, shade: .shade50))
            static let range5 = UIColor.muriel(color: MurielColor(name: .airo, shade: .shade70))
            static let selectedDay = UIColor.accent(shade: .shade50)
        }
```

These may not be the best names, but the idea is to leave the color definitions in `WPStyleGuide.Stats` so they can still be changed in one location and all usage will be updated accordingly.

Reply:
That's perfect, @ScoutHarris 😍